### PR TITLE
adding utility that extracts interfile+binary from SPECT dicom data

### PR DESCRIPTION
**Pull request by danieldeidda**

### CODE REVIEW

File: src/utilities/CMakeLists.txt [hunk 1]
```diff
@@ -73,6 +73,7 @@ set(${dir_EXE_SOURCES}
         write_sinogram_to_txt.cxx
         find_sum_projection_of_viewgram_and_sinogram.cxx
         separate_true_from_random_scatter_for_necr.cxx
+        SPECT_dicom_to_interfile.cxx
```
Comment (KrisThielemans) [inline src/utilities/CMakeLists.txt:76]: probably within `HAVE_ITK` or so

File: src/utilities/SPECT_dicom_to_interfile.cxx [hunk 2]
```diff
@@ -0,0 +1,505 @@
+#include <iostream>
```
Comment (KrisThielemans) [inline src/utilities/SPECT_dicom_to_interfile.cxx:1]: add usual copyright header

**Review by KrisThielemans — CHANGES_REQUESTED** [review]
will need to add it to release notes

**Comment by KrisThielemans** [comment]
@danieldeidda how does this handle multiple energy window data. Is there one DICOM file per energy window?

**Comment by danieldeidda** [comment]
I have never seen a case with multiple energy windows data in the same file in our case we usually have different dicom files but I am pretty sure the dicom file has the information if the acquisition was performed with multiple energy windows

**Comment by KrisThielemans** [comment]
Replaced by #1182

**Pull request closed without merging**